server/postgresSeed.js

```diff
@@ -7,9 +8,25 @@ const pool = new Pool({
   port: process.env.PG_PORT
 });
 
-pool
-  .query('SELECT NOW() as now')
-  .then(res => console.log(res))
-  .catch(e => console.error(e.stack))
+// pool
+//   .query('SELECT NOW() as now')
+//   .then(res => console.log(res))
+//   .catch(e => console.error(e.stack))
```

Comment:
if you are done with these lines, then maybe you can remove them? it's up to you